client/verta/verta/_registry/entity_with_workspace_service.py

```diff
@@ -0,0 +1,36 @@
+from verta._internal_utils import _utils
+from verta._tracking.entity import _ModelDBEntity
+
+
+class _ModelDBEntityWithWorkspaceService(_ModelDBEntity):
```

Comment:
Could we change this class and file name to `_ModelDBRegistryEntity` so it's more extensible?

I know `_ModelDBEntityWithWorkspaceService` is more specific, but I don't think future developers will know how to use it.

Reply:
Yes, I like your idea better. Renamed it

Reply:
Thank you